build.sh

```diff
@@ -7,7 +7,7 @@ ANTLR_JAR="$ANTLR_HOME/$ANTLR"
 alias antlr4="java -jar $ANTLR_JAR"
 
 # antlr4 gen
-antlr4 -Werror -Dlanguage=Cpp -listener -visitor src/parsing/DLLexer.g4 src/parsing/DLParser.g4 -o gen
+antlr4 -Werror -Dlanguage=Cpp -listener -visitor lib/antlr/DLLexer.g4 lib/antlr/DLParser.g4 -o gen
```

Comment:
Maybe we should specify antlr dir path for .tokens file?
```
antlr4 -Werror -Dlanguage=Cpp -listener -visitor -lib lib/antlr lib/antlr/DLLexer.g4 lib/antlr/DLParser.g4 -o gen
```

Reply:
> I believe that vm is not a part of lib, because we will modify it frequently.

Even though `vm` is part of core, It doesn't seem quite right to  make modifications **directly**, which makes it difficult to maintain the code. Maybe we can treat it as a module or just leaving its interface in the core?

Reply:
> Maybe we should specify antlr dir path for .tokens file?
> 
> ```
> antlr4 -Werror -Dlanguage=Cpp -listener -visitor -lib lib/antlr lib/antlr/DLLexer.g4 lib/antlr/DLParser.g4 -o gen
> ```

Forgot to add path, THANKS for the remainder.